src/components/common/Builder/BlocksBuilder/utils/validateConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { isEmpty } from "lodash-es";
```

Comment:
No need for isEmpty, `arr.length === 0` is good enough in this context/file.

Reply:
removed